python-lib/dku_google/clusters.py

```diff
@@ -109,6 +110,12 @@ def with_nodepool_labels(self, nodepool_labels=[]):
             self.nodepool_labels.update(nodepool_labels_dict)
         return self
 
+    def with_nodepool_tags(self, nodepool_tags=[]):
+        if nodepool_tags:
+            for tag in nodepool_tags:
```

Comment:
minor : the labels (just above) have some logging. Not sure it is a good idea, but it's strange to be different